src/components/MlCreatePdfForm/lib/PdfForm.tsx

```diff
@@ -76,12 +78,21 @@ const qualityOptions = [
 	},
 ];
 
-interface PdfFormProps {
+export interface PdfFormProps {
 	/**
 	 * Id of the target MapLibre instance in mapContext
 	 */
 	mapId?: string;
+	/**
+	 * Function that will be called before the PDF is created.
+	 * Allowing to access and manipulate the jspdf instance before the PDF is created.
+	 */
 	onCreatePdf?: (options: createPdfResolverParams) => createPdfResolverParams;
+	/**
+	 * sx props that will be applied to the form control components
+	 */
+	formControlStyles?: SxProps;
+	additionalFields?: React.ReactNode;
```

Comment:
Please add a description for this property.

Reply:
done